source/community/reactnative/package.json

```diff
@@ -53,8 +54,9 @@
     "adaptivecards-templating": "1.0.0-rc.0",
```

Comment:
Please upgrade dateTimePicker and Picker as well.
```
"@react-native-community/datetimepicker": "^6.0.2",
"@react-native-picker/picker": "2.4.1"
```

Reply:
@dan-arvi resolved this